packages/google_maps_flutter/google_maps_flutter_android/example/lib/readme_excerpts.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+// #docregion DisplayMode
+import 'package:google_maps_flutter_android/google_maps_flutter_android.dart';
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+
+void main() {
+  // Require Hybrid Composition mode on Android.
+  final GoogleMapsFlutterPlatform mapsImplementation =
+      GoogleMapsFlutterPlatform.instance;
+  if (mapsImplementation is GoogleMapsFlutterAndroid) {
+    mapsImplementation.useAndroidViewSurface = true;
+  }
+  // #enddocregion DisplayMode
+  runApp(const MaterialApp());
+  // #docregion DisplayMode
```

Comment:
I'm assuming this just cuts out the `runApp` line from the doc region? I didn't know you could do that. Took me a second to understand it.

Reply:
Yep, code-excerpts lets you make non-contiguous pulls, and adds (by default) a `// ...` (or whatever makes sense for the language) in the generated output.